cls/SourceControl/Git/Settings.cls

```diff
@@ -32,6 +32,15 @@ Property gitUserEmail As %String(MAXLEN = 255) [ InitialExpression = {##class(So
 /// Attribution: Whether mapped items should be read-only, preventing them from being added to source control
 Property mappedItemsReadOnly As %Boolean [ InitialExpression = {##class(SourceControl.Git.Utils).MappedItemsReadOnly()} ];
 
+/// Attribution: Whether basic mode should be enabled for user ${username}, greatly simplifying the functionality of the package, requiring no knowledge of git
```

Comment:
Nit: this is not Attribution-related

Reply:
Oops was just trying to follow the pattern of comments